File: task-03/Javascript/SubTask_3.js
const userInput=require('readline');
const Input=userInput.createInterface({
    input:process.stdin,
    output:process.stdout
});
function diamond(n) {
    n=Math.ceil(n/2);
    for(let i=1;i<=n;i++) {
        let line = '';
        for(let y=1;y<=n-i;y++) {
            line+=" ";
        }
        for(let x=1;x<2*i;x++) {
            line+="*";
        }
        console.log(line);
    }
    for(let i=n-1;i>=1;i--) {
        let line="";
        for(let y=1;y<=n-i;y++) {
            line+=" ";
        }
        for(let x=1;x<2*i;x++) {
            line+="*";
        }
        console.log(line);
    }
}
Input.question('Enter the number n: ',(input)=>{
    const n=parseInt(input);
    diamond(n);
    Input.close();
});

File: task-03/Python/SubTask_3.py
import math

def printStars(n):
    for i in range(1,n+1):
        for y in range(1,n-i+1):
            print(" ",end="")
        for x in range(1,2*i):
            print("*",end="")
        print()
    for i in range(n-1,0,-1):
        for y in range(1,n-i+1):
            print(" ",end="")
        for x in range(1,2*i):
            print("*",end="")
        print()

n=int(math.ceil(int(input("Enter the number n: "))/2))
printStars(n)